repaygithub/cactus
# CACTUS-552 :: Multiselect Order

Ticket: https://repayonline.atlassian.net/browse/CACTUS-552

### Testing
1. Bring up storybook and go to the Select/With Multiselect story
2. Clear out all of the values in the multiselect
3. Select options in reverse order compared to the order they're listed in the options. For example, select "Bisbee", then select "Avondale".
4. Make sure the options are displayed in the order you chose them, regardless of what order they appear in the options list.
5. Go through each of the other Select stories and make sure you're still able to select values for a normal select, a combobox, and multiselect with combobox. Make sure the order is maintained for multiselect/combobox when you create new options as well.
